psalm.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
```

Comment:
I've never worked with psalm before, so I'm wondering where's this config coming from? Is that like a recommended set of checks?

Reply:
This file is the default configuration when you run `psalm --init src/ 8`. With level 8, most of rules are marked as information, not errors.
Then I progessively raised the level by removing those info rules.